management/server/setupkey.go

```diff
@@ -83,24 +127,25 @@ func (key *SetupKey) IsOverUsed() bool {
 }
 
 // GenerateSetupKey generates a new setup key
-func GenerateSetupKey(name string, t SetupKeyType, validFor time.Duration) *SetupKey {
+func GenerateSetupKey(name string, t SetupKeyType, validFor time.Duration, autoGroups []string) *SetupKey {
```

Comment:
we could keep some of these setup key functions and methods private, what do you think?

Reply:
Good idea, but it is used somewhere and won't be available anymore.
I think that we should leave for the major refactor of the account manager. There are many functions to remove as well.